### PR TITLE
Task/APPS-1703 — update individual product pages

### PR DESCRIPTION
**Pull request by giuliacampana**
PCM:
Screen Shot 2020-01-14 at 11 45 18 AM
Screen Shot 2020-01-14 at 11 45 28 AM
Screen Shot 2020-01-14 at 11 45 38 AM

CARE:
Screen Shot 2020-01-14 at 11 45 47 AM
Screen Shot 2020-01-14 at 11 45 55 AM
Screen Shot 2020-01-14 at 11 46 04 AM

### CODE REVIEW

File: src/containers/products/styled/StyledProductComponents.js [hunk 1]
```diff
@@ -233,83 +201,84 @@ const ProductOverviewDescriptionWrapper = styled.span`
...
+
+  @media only screen and (max-width: ${MEDIA_QUERY_MD}px) {
+    margin-top: 32px;
+  }
 
   @media only screen and (min-width: ${MEDIA_QUERY_LG}px) {
-    top: auto;
+    align-items: flex-start;
+    flex-direction: column;
+    margin: 0 0 200px 0;
```
Comment (anzioka) [inline src/containers/products/styled/StyledProductComponents.js]: `margin: 0 200px 0`

File: src/containers/products/styled/StyledProductComponents.js [hunk 2]
```diff
@@ -233,83 +201,84 @@ const ProductOverviewDescriptionWrapper = styled.span`
...
+  @media only screen and (max-width: ${MEDIA_QUERY_MD}px) {
+    margin-top: 300px;
   }
 
   @media only screen and (min-width: ${MEDIA_QUERY_LG}px) {
-    font-size: 36px;
+    align-items: flex-start;
+    flex-direction: row;
+    margin: 440px auto 96px auto;
+    width: ${CONTENT_WIDTH}px;
```
Comment (anzioka) [inline src/containers/products/styled/StyledProductComponents.js]: `margin: 440px auto 96px`

File: src/containers/products/CareProductPage.js [hunk 3]
```diff
@@ -2,146 +2,158 @@
...
+  MEDIA_QUERY_LG,
+  MEDIA_QUERY_MD,
+  MEDIA_QUERY_TECH_SM,
+} from '../../core/style/Sizes';
+import { NEUTRALS } from '../../core/style/Colors';
+
+const IntroContent = styled(Content)`
+  @media only screen and (min-width: ${MEDIA_QUERY_LG}px) {
+    margin: 70px auto 451px auto;
+  }
```
Comment (anzioka) [inline src/containers/products/CareProductPage.js]: `margin: 70px auto 451px;`

File: src/containers/products/styled/StyledProductComponents.js (hunk 1 above)
Comment (giuliacampana) [reply src/containers/products/styled/StyledProductComponents.js]: actually i think it should be `margin: 0 0 200px;`. but good call!

[the remark this answers is not in the document]
Comment (giuliacampana) [reply src/containers/products/styled/StyledProductComponents.js]: oh wait i just saw this one lol

**Pull request merged**